website/preprints/tasks.py

```diff
@@ -62,3 +63,24 @@ def update_or_enqueue_on_preprint_updated(preprint_id, saved_fields=None):
             },
             celery=True
         )
+
+
+@celery_app.task(ignore_results=True, max_retries=5, default_retry_delay=CROSSREF_FAIL_RETRY_DELAY)
```

Comment:
Need to add a `bind=True` to this.  https://stackoverflow.com/questions/54899320/what-is-the-meaning-of-bind-true-keyword-in-celery

Reply:
Also please add in `acks_late=True` to the decorator.

Reply:
Got this on merge